lib/report_formatter/timeline.rb

```diff
@@ -87,7 +87,13 @@ def tl_event(tl_xml, row, col)
         when "BottleneckEvent"
           #         e_title = "#{ui_lookup(:model=>rec[:resource_type])}: #{rec[:resource_name]}"
           e_title = rec[:resource_name]
-          e_image = "/images/icons/new/#{bubble_icon(rec[:resource_type])}.png"
+          e_image_name = if rec[:resource_type].eql?("ExtManagementSystem") &&
```

Comment:
Also, here, if rec is the object, then this just becomes `rec.resource.kind_of?(ExtManagementSystem)`